crates/providers/src/new.rs

```diff
@@ -108,13 +276,12 @@ pub trait Provider<N: Network, T: Transport + Clone = BoxTransport>: Send + Sync
         &self,
         address: Address,
         tag: Option<BlockId>,
-    ) -> TransportResult<alloy_primitives::U256> {
+    ) -> TransportResult<U256> {
         self.client().prepare("eth_getTransactionCount", (address, tag.unwrap_or_default())).await
     }
 
     /// Get a block by its number.
-    ///
-    /// TODO: Network associate
+    // TODO: Network associate
```

Comment:
not a blocker

Reply:
There is no network block RPC type

Reply:
https://github.com/alloy-rs/alloy/issues/289